engine/installation/linux/debian.md

```diff
@@ -110,12 +112,23 @@ Docker from the repository.
     the last line.
     **Do not use these unstable repositories on production systems or for non-testing workloads.**
 
-    ```bash
-    $ sudo add-apt-repository \
-           "deb https://apt.dockerproject.org/repo/ \
-           debian-$(lsb_release -cs) \
-           main"
-    ```
+    - **Debian**:
+
+      ```bash
+      $ sudo add-apt-repository \
+             "deb https://apt.dockerproject.org/repo/ \
+             debian-$(lsb_release -cs) \
```

Comment:
@mstanleyjones this requires installing a package called `lsb-release` and 24.5MB of other crud. Is there something else we can do, eg. mucking with `cat /etc/*-release`.